views/HakaiChlorophyllSampleProvisional.sql

```diff
@@ -0,0 +1,162 @@
+DROP TABLE IF EXISTS erddap."HakaiChlorophyllSampleResearch";
+CREATE TABLE IF NOT EXISTS erddap."HakaiChlorophyllSampleResearch" AS
+SELECT "work_area",
+    "organization",
+    "survey",
+    "site_id",
+    "lat",
+    "long",
+    "gather_lat",
+    "gather_long",
+    "line_out_depth",
+    "pressure_transducer_depth",
+    "collected",
+    -- phaeo
+    -- There are a few cases where records have multiple nil
+    (array_remove(array_agg(phaeo_20um), Null)) [1]::NUMERIC phaeo_20um,
+    (array_remove(array_agg(phaeo_3um), Null)) [1]::NUMERIC phaeo_3um,
+    (array_remove(array_agg(phaeo_2um), Null)) [1]::NUMERIC phaeo_2um,
+    (array_remove(array_agg(phaeo_gf_f), Null)) [1]::NUMERIC phaeo_gf_f,
+    (array_remove(array_agg(phaeo_bulk_gf_f), Null)) [1]::NUMERIC phaeo_bulk_gf_f,
+    -- chla
+    (array_remove(array_agg(chla_20um), Null)) [1]::NUMERIC chla_20um,
+    (array_remove(array_agg(chla_3um), Null)) [1]::NUMERIC chla_3um,
+    (array_remove(array_agg(chla_2um), Null)) [1]::NUMERIC chla_2um,
+    (array_remove(array_agg(chla_gf_f), Null)) [1]::NUMERIC chla_gf_f,
+    (array_remove(array_agg(chla_bulk_gf_f), Null)) [1]::NUMERIC chla_bulk_gf_f
+    -- phaeo_flag
+    (array_remove(array_agg(phaeo_20um_flag), Null)) [1]::NUMERIC phaeo_20um_flag,
+    (array_remove(array_agg(phaeo_3um_flag), Null)) [1]::NUMERIC phaeo_3um_flag,
+    (array_remove(array_agg(phaeo_2um_flag), Null)) [1]::NUMERIC phaeo_2um_flag,
+    (array_remove(array_agg(phaeo_gf_f_flag), Null)) [1]::NUMERIC phaeo_gf_f_flag,
+    (array_remove(array_agg(phaeo_bulk_gf_f_flag), Null)) [1]::NUMERIC phaeo_bulk_gf_f_flag,
+    -- chla
+    (array_remove(array_agg(chla_20um_flag), Null)) [1]::NUMERIC chla_20um_flag,
+    (array_remove(array_agg(chla_3um_flag), Null)) [1]::NUMERIC chla_3um_flag,
+    (array_remove(array_agg(chla_2um_flag), Null)) [1]::NUMERIC chla_2um_flag,
+    (array_remove(array_agg(chla_gf_f_flag), Null)) [1]::NUMERIC chla_gf_f_flag,
+    (array_remove(array_agg(chla_bulk_gf_f_flag), Null)) [1]::NUMERIC chla_bulk_gf_f_flag
```

Comment:
these flag columns should have `::TEXT` instead of `::NUMERIC`

Reply:
sounds good